### DISCUSSION TITLE
TTS Sounds like a chipmunk :chipmunk:

### DISCUSSION THREAD

**Discussion by Cadair** [BigBobbas/ESP32-S3-Box3-Custom-ESPHome#80] [Q&A] [answered]
Thanks so much for this config, it's an epic starting point to customise these devices.

I have just flashed the version on main and also the dev version, and both suffer from an issue where the TTS audio sounds like it's been sped up a lot. Regular music playback is fine however.

Any suggestions anyone has would be awesome.

**Comment by BigBobbas** [comment, marked as answer]
Hi there 
you can change the sample rate for the TTS service that you are using. details can be found in this post  https://github.com/BigBobbas/ESP32-S3-Box3-Custom-ESPHome/issues/48#issuecomment-2235138548

Hope this helps - a google should help find which rate to use for the TTS you are using 👍 
Cheers 

Rob

**Reply by Cadair** [reply]
Thanks a lot!

**Comment by jckh1** [comment]
Hi guys
I have similar issue. I have tried various sampling rates but still cannot fix it. Could you share what is the correct value should I use?

**Reply by BigBobbas** [reply]
Hi , which service are you using for tts ?

**Reply by jckh1** [reply]
Thank you Bob for getting me back on this issue.

I was trying this TTS. Here is my sample:

![TTS-screenshot](https://github.com/user-attachments/assets/7a81983f-15af-4764-b6c2-eeecd641f696)

**Reply by BigBobbas** [reply]
hi the sample rate for piper needs to be set to 16000 and hopefully that should play at the correct speed.
once changed in the config you need to click save and then install
best wishes
Rob 👍

**Reply by jckh1** [reply]
Thanks for the prompt reply. I managed to fix the TTS.Speak issue after setting sampling rate to 16000 but encountered another problem. The voice response from the Voice Assist becomes slow and the Amy Medium voice sounds like male. Please see my sample video here.

https://github.com/user-attachments/assets/16a543aa-6fd2-4e50-89a4-0149877fe4e9

**Reply by youkorr** [reply]
I left mine at 24000 16000 it's slow

**Reply by youkorr** [reply]
what voice did you take

**Reply by jckh1** [reply]
I am using Amy Medium voice. If I set to 24000, it is perfect on Voice Assist but sounds like chipmunk on TTS.Speak. When I set it to 16000, TTS.Speak is perfect but Voice Assist is slow and sounds like old man.... 😅

**Reply by BigBobbas** [reply]
thats interesting and just looked and it seems that .speak doesnt use piper when using it from dev tools. If you go to 'media' in HA then select tts you will see 2 options, i'm not sure how to change from one to the other from dev tools,

**Comment by youkorr** [comment]
you should find a voice that matches

**Reply by jckh1** [reply]
What is your recommended voice?

**Comment by youkorr** [comment]
Personally I use Google Generative AI

**Reply by jckh1** [reply]
Yes, me too....

**Comment by youkorr** [comment]
I can ask him several requests at the same time

**Comment by youkorr** [comment]
don't forget that your box is LVGL compatible

**Comment by jckh1** [comment]
Okay, I think I have resolved the issue by choosing the low quality voices. My current settings are en_US Amy (low) or Danny (low) voice and the sampling rate is 16000.
